src/main/java/com/github/dadrus/App.java

```diff
@@ -6,12 +6,13 @@
  */
 public class App 
 {
+	public static final String PASSWORD="foo"; // just to see the sonar results in a pr
     public static void main( String[] args )
     {
         System.out.println( "Hello World!" );
     }
 	
 	public void test() throws Exception {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)